.github/workflows/tests.yml

```diff
@@ -0,0 +1,30 @@
+name: rubocop-govuk
+
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Install Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        ruby-version: 2.6
+
+    - uses: actions/cache@v1
+      with:
+        path: vendor/bundle
+        key: ${{ runner.os }}-gems-${{ hashFiles('**/Gemfile.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-gems-
+
+    - name: Install dependencies
+      run: |
+        gem install bundler
+        bundle config path vendor/bundle
```

Comment:
Is this needed? Just wondering if it'd get repetitive across lots of repos.

Reply:
I was having a play with this. We can [make it quite a lot shorter](https://github.com/alphagov/rubocop-govuk/pull/45/files). The gem installation is really fast, so I'm wondering if it's worth the hassle. Perhaps it's only necessary for repos with lots of gems.

Reply:
It's needed if we're making a bunch of changes in short succession, otherwise Actions `bundle install`s all the gems from scratch on every run - which took minutes on the coronavirus forms. But for a low-traffic app like this, maybe not so much. When we use this in more places, there are definitely opportunities for refactoring.

Reply:
Yeah. The `name` keys are there to describe what's going on. But I suppose that's super obvious from what the commands are doing. Feel free to push commits to this branch, both of you!

Reply:
(And then report back on the RFC if you think Actions is ❤️or 😭.)